app/views/api_docs/vendor_api_docs/pages/home.md

```diff
@@ -15,6 +15,17 @@ Providers can then use the API for:
 
 To get an idea of how the API works, we recommend you [review the example usage scenarios](/api-docs/usage-scenarios).
 
+## What the API doesn't support
+
+The API currently doesn't support the following features:
+- Application Deferrals
```

Comment:
```suggestion
- Deferring offers
```

(deferring an application is slightly different, and we don't support it anywhere)